apps/studio/prisma/generate.cts

```diff
@@ -37,7 +37,7 @@ const HEADER = [
   ts.addSyntheticLeadingComment(
     KYSELY_IMPORT,
     ts.SyntaxKind.SingleLineCommentTrivia,
-    "THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT IT MANUALLY.",
+    " THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT IT MANUALLY.",
```

Comment:
nit: why is there a leading space?

Reply:
so the comment looks nice haha. see https://github.com/opengovsg/isomer/pull/667/files#diff-23e6bf6b54d1b3785fa9d5c8684ad931e5cc7eeab03578e8db8ca75344071b0a